Makefile

```diff
@@ -117,12 +115,9 @@ venv/bin/activate: tests/requirements.txt
 	touch venv/bin/activate
 
 test-docker-utils:
-	mkdir -p ../debian/base/include/etc/confluent/docker
-	cd java \
-	&& mvn clean compile package assembly:single \
-	&& src/test/bin/cli-test.sh \
```

Comment:
I couldn't find this script, so I guess it wasn't doing anything. Also wasn't sure if we are running this target as part of a jenkins job somewhere so that it gets tested periodically. Hence removed. Looks like the target is currently set to be called manually.

Reply:
Looks like @aayars removed it in https://github.com/confluentinc/cp-docker-images/pull/365/files?utf8=%E2%9C%93&diff=unified#diff-0b396f5c6040af34af499cfc1a7f0737 so he or @arrawatia may have more context on the need for it or if it is ok to remove.